.github/workflows/ci.yml

```diff
@@ -112,20 +107,26 @@ jobs:
       # Enable corepack for packageManager config
       - run: corepack enable || sudo corepack enable
       - run: yarn install
+      # Set up docker-bake files used by docker/bake-action
+      - run: node_modules/.bin/synthetic-chain prepare-ci
 
-      - name: build test images ${{ matrix.platform }} == ${{ env.DEFAULT_PLATFORM }}
-        run: |
-          docker info
-          node_modules/.bin/synthetic-chain build ${{ matrix.platform == env.DEFAULT_PLATFORM && ' ' || '--dry' }}
-      - name: run test images
-        if: ${{ matrix.platform == env.DEFAULT_PLATFORM }}
-        run: node_modules/.bin/synthetic-chain test
+      # Matrix within "bake" instead of GitHub Actions because the actions
+      # won't resolv the DAG of what to build first.
+      - name: Push proposal "use" images
+        uses: docker/bake-action@v4
```

Comment:
> This isn't meant to push tags for "use" images out of PR builds. It does for review but once the PR is approved I'll change it to only push from the default branch. That sidesteps the issue of distinguishing use images by branch origin.

Well that answers my question. If we could figure out prefixing, it might be nice to avoid overriding, at least for the case where we manually trigger a workflow.

Reply:
It's not. I covered that in the PR description.

> This isn't meant to push tags for "use" images out of PR builds. It does for review but once the PR is approved I'll change it to only push from the default branch. That sidesteps the issue of distinguishing use images by branch origin.

Reply:
Agree it "might be nice" . I don't think we need it to deliver #25. Let's tackle if a need presents itself.